types/node-red__util/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "private": true,
+    "dependencies": {
```

Comment:
Can you add this entry back? `jsonata`. Without that the PR cannot be verified locally (I understeand it will fail with whitelist requirement)

Reply:
Thanks for the heads up! Forgot to add it back after running local tests. Fixed now.